app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+import os
+
+
+def move_file(command: str):
+    parsed = command.split()
+
+    if parsed[0] == "mv" and parsed[1] != parsed[2]:
```

Comment:
If you use `parsed[0]` / `parsed[1]` /  `parsed[2]` several times - save it to the variables.